application-admintools-ui/src/main/resources/AdminTools/Code/Configuration.xml

```diff
@@ -105,7 +105,7 @@
       <serverLocation/>
     </property>
     <property>
-      <spamSize>2</spamSize>
+      <spamSize>50</spamSize>
```

Comment:
Why is this `spamSize` field doing?

Reply:
It's a leftover from testing that was released. I was using a comments spam size of 2, but the default should be at 50.